vendor/deps/cli-kit/lib/cli/kit/resolver.rb

```diff
@@ -29,6 +29,17 @@ def command_not_found(name)
           $stderr.puts(CLI::UI.fmt("{{command:#{@tool_name} #{name}}} was not found"))
         end
 
+        if ShopifyCli::Project.has_current?
+          $stderr.puts(CLI::UI.fmt(<<~INFO
+            Project: {{green:#{ShopifyCli::Project.project_name}}} (#{ShopifyCli::Project.current_project_type})
+            Run {{command:shopify help}} to see available commands for #{ShopifyCli::Project.current_project_type} projects.
+            
+            INFO
+          ))
+        else
+          $stderr.puts(CLI::UI.fmt("No project found. Run {{command:shopify help}} to see available core commands.\n\n"))
```

Comment:
I wonder if instead of saying `No project found`, it might be more helpful to say `You are not in a project folder`, or something to that effect?